### PR TITLE
Refactoring and fix for persisted payment_sources

### PR DESCRIPTION
**Pull request by cesargdm**

### CODE REVIEW

**Comment by cesargdm** [comment]
The main change is the fix of a bug that persisted the payment methods of consulted clients.

In the case of the added files, it's a simple refactor, I splitted the _resources_ objects, such as Card, Charge, Customer, etc. to a new `resources` directory, to better understanding and future support and used at your main file `conekta.js`

**Comment by eduenriquez** [comment]
@cesargdm  let me review this change with the team and in this week we possibly merge the pr thanks!

**Comment by eduenriquez** [comment]
@cesargdm running the specs i note that some of them are not passing that means that may some features will be broken if we merge this PR can you fix those specs ?

File: lib/resources/Order.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
+const Resource = require('../Resource')
+
```
Comment (eduenriquez) [inline lib/resources/Order.js]: In order to fix json order test "should return something" i think here can be 

`const { API_BASE } = require('../constants')`

as API_BASE is not sharing his value from Resource import

**Comment by cesargdm** [comment]
Ready, I fixed  4 issues, but these two are still present:

```
  1) Order Charges create should return instance object with id:
     Uncaught TypeError: Cannot read property 'hasOwnProperty' of null
      at createCharge (test/test.js:524:24)
      at Object.next (lib/resources/Order.js:120:18)
      at Object.<anonymous> (lib/Resource.js:226:18)
      at Request.request.(anonymous function) [as _callback] (lib/Requestor.js:104:12)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)

  2) Order Charges next page should return instance object with id:
     Uncaught TypeError: Cannot read property 'order_id' of null
      at createCharge (test/test.js:533:34)
      at Object.next (lib/resources/Order.js:120:18)
      at Object.<anonymous> (lib/Resource.js:226:18)
      at Request.request.(anonymous function) [as _callback] (lib/Requestor.js:104:12)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)
``` 

But take In note that I noticed that *in your master branch this issues is also present*.

**Comment by eduenriquez** [comment]
@cesargdm sure i know that 2 test are failing i will fix those in one of the following PR, one more thing can you fix those little code style snippets marked by houndci-bot ? we follow `standard`  style guide and that should be all to merge this =)

**Comment by cesargdm** [comment]
Would you be ok if I added a .eslintrc file to the repo? That would be easier for this and future maintenance commits

**Comment by eduenriquez** [comment]
The think is that we need to add a .yml and the .eslintrc to sync linter at github and locally, its ok if you only add the indicated by houndci-bot for the moment :)

**Comment by cesargdm** [comment]
Fixed code style to match Standard JS

**Comment by cesargdm** [comment]
Not sure where am I missing the semicolons, the linter that extends the standard guide doesn't output any message about missing semicolons, my .eslintrc.json configuration is: 

```javascript
{
  "extends": "standard"
}
```

And the output I get by running `npx standard` is:

```
standard: Use JavaScript Standard Style (https://standardjs.com)
  /Users/cesargdm/sites/conekta-node/lib/base64.js:15:16: 'new Buffer()' was deprecated since v6. Use 'Buffer.alloc()' or 'Buffer.from()' (use 'https://www.npmjs.com/package/safe-buffer' for '<4.5.0') instead.
  /Users/cesargdm/sites/conekta-node/test/test.js:6:7: 'base64' is assigned a value but never used.
  /Users/cesargdm/sites/conekta-node/test/test.js:10:7: 'PRODUCTION_KEY' is assigned a value but never used.
  /Users/cesargdm/sites/conekta-node/test/test.js:31:1: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:34:3: 'beforeEach' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:40:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:51:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:61:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:62:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:71:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:72:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:88:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:89:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:110:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:111:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:119:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:120:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:130:1: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:133:3: 'beforeEach' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:139:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:140:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:141:37: Use path.join() or path.resolve() instead of + to create paths.
  /Users/cesargdm/sites/conekta-node/test/test.js:146:20: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:153:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:154:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:158:10: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:165:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:166:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:167:19: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:174:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:175:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:176:19: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:177:15: 'order' is assigned a value but never used.
  /Users/cesargdm/sites/conekta-node/test/test.js:181:12: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:189:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:190:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:212:10: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:213:23: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:221:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:222:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:223:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:224:21: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:233:14: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:242:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:244:19: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:260:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:261:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:262:24: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:263:45: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:273:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:274:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:276:24: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:283:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:284:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:285:24: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:286:45: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:292:16: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:301:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:302:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:303:24: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:304:45: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:305:44: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:315:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:317:19: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:328:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:329:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:330:23: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:331:45: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:341:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:342:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:343:23: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:350:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:351:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:352:23: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:353:47: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:356:16: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:365:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:366:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:367:23: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:368:47: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:369:41: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:379:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:381:19: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:394:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:395:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:396:28: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:397:45: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:407:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:408:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:409:28: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:416:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:417:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:418:28: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:419:47: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:423:16: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:432:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:433:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:434:28: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:435:47: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:436:46: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:446:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:448:19: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:457:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:458:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:459:28: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:460:45: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:470:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:471:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:472:28: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:479:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:480:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:481:28: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:482:47: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:485:16: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:494:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:495:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:496:28: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:497:47: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:498:46: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:508:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:510:19: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:521:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:522:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:523:22: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:530:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:531:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:532:22: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:533:44: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:544:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:567:10: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:574:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:575:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:576:22: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:585:1: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:588:3: 'beforeEach' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:594:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:595:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:596:31: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:604:1: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:607:3: 'beforeEach' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:613:7: 'createdPlan' is assigned a value but never used.
  /Users/cesargdm/sites/conekta-node/test/test.js:614:3: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:615:43: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:620:3: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:630:8: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:637:3: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:638:40: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:644:3: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:645:40: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:646:19: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:654:1: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:657:3: 'beforeEach' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:665:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:666:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:670:10: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:679:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:680:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:690:10: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:691:9: 'res' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:692:16: 'res' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:698:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:699:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:700:39: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:704:12: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:712:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:713:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:714:34: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:720:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:721:30: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:728:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:729:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:730:39: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:737:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:738:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:739:39: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:743:12: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:751:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:752:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:753:39: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:756:12: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:764:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:765:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:766:39: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:770:12: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:778:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:779:5: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:780:39: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:781:20: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:789:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:792:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:793:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:803:12: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:817:14: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:826:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:827:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:828:58: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:842:14: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:850:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:851:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:852:58: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:853:48: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:862:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:863:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:864:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:874:12: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:875:39: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:888:16: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:897:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:898:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:908:12: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:909:50: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:918:3: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:921:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:922:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:933:12: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:937:14: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:945:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:946:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:947:55: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:948:39: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:956:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:957:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:958:55: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:961:14: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:962:42: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:971:5: 'describe' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:972:7: 'it' is not defined.
  /Users/cesargdm/sites/conekta-node/test/test.js:973:48: Expected error to be handled.
  /Users/cesargdm/sites/conekta-node/test/test.js:974:29: Expected '===' and instead saw '=='.
```

**Comment by cesargdm** [comment]
@eduenriquez so, is everything ok?

**Comment by eduenriquez** [comment]
@cesargdm sorry for no attend, ok makes sense , i thought houndci-bot was configured by default with standard linter, i will merge this right now and the next until this week i update npm repo

**Comment by eduenriquez** [comment]
BTW thanks a lot for this contribution 🎉

**Pull request merged**